packages/listr2/src/utils/concurrency.ts

```diff
@@ -2,6 +2,10 @@
 // with MIT license, since it is again ESM only
 // we needed to stop on the first failing case unlike the original
 export class Concurrency {
+  promise: Promise<void>
```

Comment:
Just that linting rules should be setup like that or for consistency with all the other things can you mark this with `public` marker.